index.html

```diff
@@ -161,6 +162,7 @@ <h1 class="w-full my-2 text-4xl font-bold leading-tight text-center text-gray-80
           <div class="h-1 mx-auto gradient w-64 opacity-25 my-0 py-0 rounded-t"></div>
         </div>
 
+        <div class="card">
```

Comment:
What do you think about [Pointer Events](https://tailwindcss.com/docs/pointer-events) ? I think we should keep the Tailwind's class.

Reply:
Like 'cursor-pointer' class

Reply:
Yeah, I think we should keep the Tailwind

Reply:
Sounds good! Should we use the `cursor-pointer` class, then?